build.gradle

```diff
@@ -186,6 +192,10 @@ subprojects { project ->
   def platformJavaHome = System.getProperty('test.java.home')
 
   test {
+    javaLauncher = javaToolchains.launcherFor {
+      languageVersion = JavaLanguageVersion.of(14)
```

Comment:
Will allow overrides for this, defaulting to 11, and fix up CI to use it instead of requiring circleci images to select